locale/es/site.json

```diff
@@ -88,22 +99,30 @@
     "guides": {
       "link": "docs/guides",
       "text": "Guías"
+    },
+    "dependencies": {
+      "link": "docs/meta/topics/dependencies",
+      "text": "Dependencias"
     }
   },
   "getinvolved": {
     "link": "get-involved",
-    "text": "Participa",
```

Comment:
I think "Participa" makes more sense in this context rather than "Implícate"

Reply:
I tried it with 'implícate' cause is the word used at catalan translation. I can change in both files by 'Participa' to be consistent. 
Thanks @academo